internal/oci/runtime_oci.go

```diff
@@ -87,12 +87,11 @@ func (r *runtimeOCI) CreateContainer(c *Container, cgroupParent string) (err err
 	defer parentStartPipe.Close()
 
 	var args []string
-	if r.config.CgroupManager == SystemdCgroupsManager {
+	if r.config.CgroupManager().IsSystemd() {
 		args = append(args, "-s")
 	}
-	if r.config.CgroupManager == CgroupfsCgroupsManager {
-		args = append(args, "--syslog")
-	}
+
+	args = append(args, "--syslog")
```

Comment:
Not sure why it was conditional before and is now it's not.

Reply:
I made the switch because I couldn't tell why it was conditional and it made no sense to have it so. it eventually got squashed into everything else.